scripts/convert-color.ts

```
@@ -0,0 +1,86 @@
+
+function loadJSON(filePath: string) {
+  return JSON.parse(fs.readFileSync(path.resolve(filePath), 'utf-8'));
+}
+
+// 원시 토큰 로드
+const primitiveLight = loadJSON('tokens/primitive-light.json');
+const semanticLight = loadJSON('tokens/semantic-light.json');
+const primitiveDark = loadJSON('tokens/primitive-dark.json');
+const semanticDark = loadJSON('tokens/semantic-dark.json');
+
+function flatten(obj: any, prefix = '', result: Record<string, any> = {}) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)